web/wp-content/mu-plugins/pantheon/pantheon-login-form-mods.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Modify the WordPress login form for Pantheon
+ */
+
+ /**
+  * Should we proceed with adding the
+  * return to Pantheon button?
+  *
+  * Only if:
+  * We are on a Pantheon subdomain and
+  * "RETURN_TO_PANTHEON_BUTTON" is not false
```

Comment:
:warning: **Warning**: Found precision alignment of 1 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).